src/omg-wallet.js

```diff
@@ -71,6 +71,23 @@ function newAddress (password) {
   })
 }
 
+function getPrivateKeys() {
+  var addresses = globalKeystore.getAddresses()
+  let password = prompt('Enter password', 'Password')
+
+  globalKeystore.keyFromPassword(password, function (err, pwDerivedKey) {
+    if (err) {
+      console.error(err)
+      return
+    }
+    // Show the first 2 addresses in the wallet
+    for(i=0;i<addresses.length;i++) {
```

Comment:
Don't you need a `var` or `let` in here e.g.`let i = 0`?

Reply:
huh, now that you mentioned it I should have needed, but it ran fine. I will add that just in case anyways